examples/practice-typescript/src/components/SideBarMenu/SideBarMenu.tsx

```diff
@@ -0,0 +1,28 @@
+import SideBarMenuItem, {
+  SideBarMenuItemProps,
+} from 'components/SideBarMenuItem/SideBarMenuItem';
+import './sideBarMenu.scss';
+
+interface SideBarMenuProps {
+  sideBarMenuItemProps: SideBarMenuItemProps[];
+}
+
+const SideBarMenu = ({ sideBarMenuItemProps }: SideBarMenuProps) => {
+  return (
+    <ul className="aside__list">
+      {sideBarMenuItemProps.map((item) => (
+        <li>
+          <SideBarMenuItem
```

Comment:
Why `SideBarMenuItem` nested in the `li`. It should be `<li>`, right?

Reply:
Fixed